build/azure-pipelines.yml

```
@@ -351,137 +345,160 @@ stages:
+          # Custom Umbraco settings
+          UMBRACO__CMS__GLOBAL__INSTALLMISSINGDATABASE: true
+          UMBRACO__CMS__GLOBAL__ID: 00000000-0000-0000-0000-000000000042
+          UMBRACO__CMS__GLOBAL__VERSIONCHECKPERIOD: 0
+          UMBRACO__CMS__GLOBAL__USEHTTPS: true
+          UMBRACO__CMS__HEALTHCHECKS__NOTIFICATION__ENABLED: false
+          UMBRACO__CMS__KEEPALIVE__DISABLEKEEPALIVETASK: true
+          UMBRACO__CMS__WEBROUTING__UMBRACOAPPLICATIONURL: https://localhost:44331/
+          ASPNETCORE_URLS: https://localhost:44331
         strategy:
           matrix:
-            Linux:
+            'Linux (SQLite)':
               vmImage: 'ubuntu-latest'
-              dockerfile: umbraco-linux.docker
-              dockerImageName: umbraco-linux
-            Windows:
+            'Windows (SQLite)':
+              vmImage: 'windows-latest'
+            'Windows (SQL Server)':
               vmImage: 'windows-latest'
-              DOTNET_GENERATE_ASPNET_CERTIFICATE: true # Automatically generate HTTPS development certificate on Windows
-              # Enable console logging in Release mode
-              Serilog__WriteTo__0__Name: Async
-              Serilog__WriteTo__0__Args__configure__0__Name: Console
-              # Set unattended install settings
-              Umbraco__CMS__Unattended__InstallUnattended: true
-              Umbraco__CMS__Global__InstallMissingDatabase: true
-              UmbracoDatabaseServer: (LocalDB)\MSSQLLocalDB
-              UmbracoDatabaseName: Playwright
-              ConnectionStrings__umbracoDbDSN: Server=$(UmbracoDatabaseServer);Database=$(UmbracoDatabaseName);Integrated Security=true;
-              # Custom Umbraco settings
-              Umbraco__CMS__Global__VersionCheckPeriod: 0
-              Umbraco__CMS__Global__UseHttps: true
-              Umbraco__CMS__HealthChecks__Notification__Enabled: false
-              Umbraco__CMS__KeepAlive__DisableKeepAliveTask: true
+              CONNECTIONSTRINGS__UMBRACODBDSN: Data Source=(localdb)\MSSQLLocalDB;AttachDbFilename=|DataDirectory|\Umbraco.mdf;Integrated Security=True
+              CONNECTIONSTRINGS__UMBRACODBDSN_PROVIDERNAME: Microsoft.Data.SqlClient
         pool:
           vmImage: $(vmImage)
         steps:
+          # Setup test environment
           - task: DownloadPipelineArtifact@2
-            displayName: Download nupkg
+            displayName: Download NuGet artifacts
             inputs:
               artifact: nupkg
-              path: $(Build.SourcesDirectory)/tests/Umbraco.Tests.AcceptanceTest/misc/nupkg
+              path: $(Agent.BuildDirectory)/app/nupkg
+
           - task: NodeTool@0
             displayName: Use Node.js $(nodeVersion)
             inputs:
               versionSpec: $(nodeVersion)
+
+          - task: UseDotNet@2
+            displayName: Use .NET $(dotnetVersion)
+            inputs:
+              version: $(dotnetVersion)
+              includePreviewVersions: $(dotnetIncludePreviewVersions)
```

We can also remove the .NET SDK version from the build pipeline by creating a [`global.json` file](https://learn.microsoft.com/en-us/dotnet/core/tools/global-json) in the repository root containing:

```json
{
  "sdk": {
    "version": "7.0.100",
    "rollForward": "latestFeature"
  }
}
```

And then remove the `dotnetVersion` and `dotnetIncludePreviewVersions` variables and replace all occurrences of this task to:

```suggestion
          - task: UseDotNet@2
            displayName: Use .NET SDK from global.json
            inputs:
              useGlobalJson: true
```

This will also ensure everyone that's running from source code will need to have the correct .NET SDK version installed (a minimum of the specified version and depending on the `rollForward` value, the same major/minor).

---

I've merged PR https://github.com/umbraco/Umbraco-CMS/pull/14720 into this PR, so this is now fixed as well!